src/linear_algebra/affine_space.lean

```diff
@@ -1664,10 +1656,10 @@ end affine_subspace
 
 /-- An `affine_map k V1 P1 V2 P2` is a map from `P1` to `P2` that
 induces a corresponding linear map from `V1` to `V2`. -/
-structure affine_map (k : Type*) (V1 : Type*) (P1 : Type*) (V2 : Type*) (P2 : Type*)
+structure affine_map (k : Type*) {V1 : Type*} (P1 : Type*) {V2 : Type*} (P2 : Type*)
```

Comment:
Another doc string needing updating for implicit arguments. (I may not have identified all such places in my comments.)